src/graphql.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+const https = require('https');
+
+/** Escape strings to prevent injection attacks. Other types aren't an issue. */
+const escapeArgValue = val => {
+	if ('string' === typeof (val)) {
+		return '"' + encodeURIComponent(val) + '"';
+	} else {
+		return val;
+	}
+};
+
+const argsString = args => {
+	const keys = Object.keys(args);
+	if (keys.length === 0) {
+		return '';
```

Comment:
Can you lint according to `standard`?

Reply:
Standard is very opinionated, not configurable, and not standard. 

Of the 3 linters that we previously in use I chose xo because it was quick and configurable. We can set the linting flags as you like, but standard makes some assumptions that I would consider bad style, and the fact that you can't configure it was making it inordinately difficult.

Reply:
Yeah. I use standard because I know feross, and I'd rather use his stylguide then waste time on my own. Spending time figuring out my own opinion isn't where I want to be, either. XO is similar, AFAIK - the option seems arbitrary. You can always set ignores for issues with particular files or lines - that what I do sometimes to get around standard being persnickety.